mapreduce/src/test/java/org/pentaho/hadoop/mapreduce/PentahoMapReduceIT.java

```diff
@@ -93,7 +97,7 @@ public void testMapRunnable_WordCount() throws IOException, KettleException, URI
       IntStream.rangeClosed( 1, ROWS_TO_CALCULATE ).mapToObj( value -> String.valueOf( WORDS_TO_CALCULATE ) )
         .collect( Collectors.toList() );
     if ( DEBUG_MODE ) {
```

Comment:
This should be if(log.isDebugEnabled())

Reply:
Done

Reply:
There is no such method, but I used log.isDebug()